compiler/damlc/tests/daml-test-files/SetFieldParens.EXPECTED.renamed-daml

```diff
@@ -1,7 +1,7 @@
 
 ==================== Renamer ====================
 SetFieldParens.foo : (Text -> (Int, Bool)) -> (Int, Bool)
-SetFieldParens.foo mkTup_aoK
+SetFieldParens.foo mkTup_aoU
   = (setField
-       @"_2" (not (getField @"_2" (mkTup_aoK "baz"))) (mkTup_aoK "bar"))
+       @"_2" (not (getField @"_2" (mkTup_aoU "baz"))) (mkTup_aoU "bar"))
```

Comment:
Why has this changed? Whats the significance of this?

Reply:
Fair question! this test was added in https://github.com/digital-asset/daml/pull/13603, it checks that the `setField` desugaring gets wrapped in parens. As explained in that PR, this happens in the renamer, but that means having new names in scope changes the generated labels. However, I noticed that before https://github.com/digital-asset/daml/pull/16379, this test didn't show any generated names, so I'm looking into that now.

Reply:
see https://github.com/digital-asset/daml/pull/16487, thanks for spotting this :)